java-chassis-distribution/src/release/NOTICE

```diff
@@ -3,3 +3,48 @@ Copyright 2017-2018 The Apache Software Foundation
 
 This product includes software developed at
 The Apache Software Foundation (http://www.apache.org/).
+
+
+================================================================
+Notice for Apache Commons Codec
+
+See licenses/NOTICE-apache-commons-codec
```

Comment:
Please put the the notices directory here.

Reply:
As explained in other discussion, there is no need to put up a notices directory which will increase end user burden on collecting legal information.